src/langs/rust/template/Cargo.toml

```diff
@@ -9,27 +9,32 @@ edition = "2021"
 async-channel = "2.0.0"
 libc = "0.2"
 rand = "0.8.5"
-soup3 = { version = "0.5.0", features = ["v3_4"] }
 url = "2.4.0"
-webkit6 = { version = "0.2.0", features = ["v2_42"] }
 
+[dependencies.soup3]
+version = "0.6.0"
+features = ["v3_4"]
+
+[dependencies.webkit6]
+version = "0.3.0"
+features = ["v2_42"]
 
 [dependencies.gtk]
-version = "0.7"
+version = "0.8.0"
 package = "gtk4"
 features = ["gnome_45"]
 
 [dependencies.adw]
-version = "0.5"
+version = "0.6.0"
 package = "libadwaita"
-features = ["v1_4", "gtk_v4_10"]
+features = ["v1_5"]
 
```

Comment:
Why did you remove the gtk feature? Pretty sure that's needed

Reply:
By adding the feature we enable certain gated functions. I am not 100% sure that we are using them, but there's no harm enabling the feature since we have a new enough gtk and people using Workbench might want to use that API.

Bumping gtk fails since the latest feature depends only on gtk 4.10.